src/main/webapp/static/js/validator_controller.js

```diff
@@ -6,41 +6,40 @@ angular
-					var receiverCuresGitHubUrl = 'https://api.github.com/repos/onc-healthit/2015-edition-cures-update-data/contents/Cures Update Receiver SUT Test Data';
-					var senderSvap2022GitHubUrl = 'https://api.github.com/repos/onc-healthit/2015-edition-cures-update-uscdi-v2-testdata/contents/Cures Update Svap Uscdiv2 Sender SUT Test Data';
-					var receiverSvap2022GitHubUrl = 'https://api.github.com/repos/onc-healthit/2015-edition-cures-update-uscdi-v2-testdata/contents/Cures Update Svap Uscdiv2 Receiver SUT Test Data';					
-					var senderSvap2023GitHubUrl = 'https://api.github.com/repos/onc-healthit/2015-edition-cures-update-uscdi-v3-testdata/contents/Cures Update Svap Uscdiv3 Sender SUT Test Data';
-					var receiverSvap2023GitHubUrl = 'https://api.github.com/repos/onc-healthit/2015-edition-cures-update-uscdi-v3-testdata/contents/Cures Update Svap Uscdiv3 Receiver SUT Test Data';			
+					var receiverGitHubUrl = 'https://api.github.com/repos/onc-healthit/2015-certification-ccda-testdata/contents/Receiver SUT Test Data';					
+					var senderCuresGitHubUrl = 'https://api.github.com/repos/onc-healthit/ccda-uscdi-certification-testdata/contents/uscdi-v1-testdata/Sender SUT Test Data';
+					var receiverCuresGitHubUrl = 'https://api.github.com/repos/onc-healthit/ccda-uscdi-certification-testdata/contents/uscdi-v1-testdata/Receiver SUT Test Data';
+					var senderSvap2022GitHubUrl = 'https://api.github.com/repos/onc-healthit/ccda-uscdi-certification-testdata/contents/uscdi-v2-testdata/Sender SUT Test Data';
+					var receiverSvap2022GitHubUrl = 'https://api.github.com/repos/onc-healthit/ccda-uscdi-certification-testdata/contents/uscdi-v2-testdata/Receiver SUT Test Data';					
+					var senderSvap2023GitHubUrl = 'https://api.github.com/repos/onc-healthit/ccda-uscdi-certification-testdata/contents/uscdi-v3-testdata/Sender SUT Test Data';
+					var receiverSvap2023GitHubUrl = 'https://api.github.com/repos/onc-healthit/ccda-uscdi-certification-testdata/contents/uscdi-v3-testdata/Receiver SUT Test Data';
+					var senderUscdiv4GitHubUrl = 'https://api.github.com/repos/onc-healthit/ccda-uscdi-certification-testdata/contents/uscdi-v3-testdata/Sender SUT Test Data';
+					var receiverUscdiv4GitHubUrl = 'https://api.github.com/repos/onc-healthit/ccda-uscdi-certification-testdata/contents/uscdi-v3-testdata/Receiver SUT Test Data';
+
 					var validationError;
 					var self = this;
 					
 					$scope.validationTypes = [
 						{ 
 							name: '2015 Certification',
-							curesUpdate: false,
-							svap2022: false,
-							svap2023: false
+							ccdaType: ''
 						},
 						{ 	
 							name: 'Cures Update',
-							curesUpdate: true,
-							svap2022: false,
-							svap2023: false
+							ccdaType: 'cures'
 						},
 						{ 	
 							name: 'SVAP 2022',
-							curesUpdate: false,
-							svap2022: true,
-							svap2023: false
+							ccdaType: 'svap'
```

Comment:
in the future, not this release, as too may things depend on it now this way, this should be changed from svap to uscdiv2

Reply:
James please create a ticket to be addressed in the future release. (this is for future release)